src/Surfnet/StepupRa/RaBundle/Service/InstitutionListingService.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Copyright 2019 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupRa\RaBundle\Service;
+
+use Surfnet\StepupMiddlewareClientBundle\Identity\Service\InstitutionListingService as ApiInstitutionListingService;
+use Surfnet\StepupMiddlewareClientBundle\Identity\Dto\InstitutionListing;
+
+class InstitutionListingService
+{
+    /**
+     * @var ApiInstitutionListingService
+     */
+    private $institutionListingService;
+
+    /**
+     * @param ApiInstitutionListingService $institutionListingService
+     */
+    public function __construct(
+        ApiInstitutionListingService $institutionListingService
+    ) {
+        $this->institutionListingService = $institutionListingService;
+    }
+
+    public function getAll()
+    {
+        $collection = $this->institutionListingService->getAll();
+        $listings = $collection->getElements();
+
+        $options = array_map(function (InstitutionListing $listing) {
+            return $listing->institution;
+        }, $listings);
+
+        $selectOptions = array_combine($options, $options);
+
+        return $selectOptions;
+    }
+}
```

Comment:
Missing linefeed at the end of the file. Why did the code sniffer not trigger on this?

Reply:
Because the tests will only run if it gets merged one level down